Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/FrameBufferMtl.mm

```diff
@@ -1655,9 +1655,11 @@ void RoundValueAndAdjustCorrespondingValue(float a,
 
     ContextMtl *contextMtl = mtl::GetImpl(context);
 
-    ANGLE_MTL_CHECK(contextMtl, packPixelsParams.offset <= std::numeric_limits<uint32_t>::max(),
+    ptrdiff_t packPixelsParamsOffset = packPixelsParams.offset;
+    ANGLE_MTL_CHECK(contextMtl, packPixelsParamsOffset >= 0, GL_INVALID_OPERATION);
+    ANGLE_MTL_CHECK(contextMtl, static_cast<std::make_unsigned_t<ptrdiff_t>>(packPixelsParamsOffset) <= std::numeric_limits<uint32_t>::max(),
                     GL_INVALID_OPERATION);
-    uint32_t offset = static_cast<uint32_t>(packPixelsParams.offset);
```

Comment:
> GLintptr is not used so that the driver would compare > 0.

I'm assuming that "the driver" in this context would mean "GPU driver code would compare > 0, but not the code in this file".